### PR TITLE
add an actix-web 2.0 project from community

### PR DESCRIPTION
**Pull request by danloh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ A curated list of examples related to actix.
 * [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
 * [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
 * [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
-
+* [Toplog](https://toplog.cc/) — A Rust Article/Event/Book Aggregator Powered by Actix-web 2.0 [source](https://github.com/danloh/toplog)
```
Comment (cdbattags) [inline README.md]: can we add a line break here before contribute?

Comment (danloh) [reply README.md]: sorry, my bad

**Comment by JohnTitor** [comment]
Also, you should resolve merge conflict (sorry for causing!).

**Comment by danloh** [comment]
I realized that there is already a good example on actix-web2.0, 
I can close this PR, Sorry for wasting your time to review.

**Pull request closed without merging**